tests/tt_metal/tt_metal/unit_tests_common/watcher/test_noc_sanitize.cpp

```diff
@@ -76,9 +76,14 @@ void RunTestOnCore(WatcherFixture* fixture, Device* device, CoreCoord &core, boo
         dram_buffer_size, 100, std::chrono::system_clock::now().time_since_epoch().count());
     tt_metal::detail::WriteToBuffer(input_dram_buffer, input_vec);
 
-    // Write runtime args - update to a core that doesn't exist
-    output_dram_noc_xy.x = 16;
-    output_dram_noc_xy.y = 16;
+    // Write runtime args - update to a core that doesn't exist or an improperly aligned address,
+    // depending on the flags passed in.
+    if (test_illegal_core) {
+        output_dram_noc_xy.x = 16;
+        output_dram_noc_xy.y = 16;
+    } else {
+        l1_buffer_addr += 1;
```

Comment:
test alignment is done by side effect - what about an enum or 2 flags?

Reply:
Fixed in latest commit